modules/knative-serving-CR-system-deployments.adoc

```diff
@@ -0,0 +1,40 @@
+[id="knative-serving-CR-system-deployments_{context}"]
+= Overriding system deployment configurations
+
+You can override the default configurations for some specific deployments by modifying the `deployments` spec in the `KnativeServing` custom resource (CR).
+
+Currently, overriding default configuration settings for the `replicas`, `labels`, `annotations`, and `nodeSelector` fields are supported.
+
+The following `KnativeServing` CR example overrides the `webhook` deployment so that:
```

Comment:
Scans a little strangely to me, but could just be me. 

Might try:

"In the following example, a `KnativeServing` CR overrides the `webhook` deployment so that:"